src/dmd/e2ir.d

```diff
@@ -3837,7 +3918,17 @@ elem *toElem(Expression e, IRState *irs)
                     }
                 }
             }
-            elem *ecall = callfunc(ce.loc, irs, ce.directcall, ce.type, ec, ectype, fd, t1, ehidden, ce.arguments);
+            elem *ethis2 = null;
+            if (ce.vthis2)
```

Comment:
here ast and elem tree are mixed.

Reply:
same reason as the other case. see my other reply.